dts/bindings/i2c/ite,it8xxx2-i2c.yaml

```diff
@@ -33,3 +30,10 @@ properties:
       description: Configuration of I2C SCL and SDA pinmux controller.
                    The SCL pin must be specified first and the SDA pin
                    second in the pinctrl-0 array.
+
+    clock-gate-offset:
+      type: int
+      required: true
+      description: |
+        The clock gate offsets combine the register offset from
+        ECPM_BASE and the mask within that register into one value.
```

Comment:
It seems like the following properties are common to both I2C controllers:

- reg
- interrupts
- port-num
- gpio-dev
- pinctrl-0
- clock-gate-offset

Only prescale-scl-low is unique to the 'enhanced' driver. So for maintainability reasons you may want to consider using `include` as is done here for example:

https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/i2c/nordic%2Cnrf-twi-common.yaml
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/i2c/nordic%2Cnrf-twi.yaml
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/i2c/nordic%2Cnrf-twim.yaml

This is not a blocking request, though, just something for you to consider. Thanks for the patch!

Reply:
Yes, It can be better maintained, thanks for your advise. Done.